lib/utils/sanitization-utils.js

```diff
@@ -24,17 +24,35 @@ function getProtocol(url) {
     if (typeof url === 'string') {
       protocol = nodeURL.parse(url).protocol;
     }
-    return (protocol === null) ? ':' : protocol;
+    return protocol;
   } else {
-    throw new Error('Mobiledoc DOM Renderer is unable to sanitize href tags in this environment');
+    // unknown env where we cannot use a pre-existing url parser,
+    // such as Fastboot's VM Sandbox
```

Comment:
@mixonic no, as mentioned in the description of this PR, it doesn't. The sandbox has `window` and `module` defined.